### PR TITLE
Add lightmode 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is only a dark theme. #21 

**What is the new behavior (if this is a feature change)?**
- A light theme has been added. #21 

**Other information**:
- You can switch between modes in the footer

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks fine to me

File: src/Routes.js [hunk 1]
```diff
@@ -20,11 +20,27 @@ class Routes extends React.Component {
...
+        <Route
+        exact
+        path='/news/:label'
+        component={(props) => <BlogListPageLabel globalStore={this.props} {...props} />}
+        />
+        <Route
+        exact
+        path='/news/article/:id'
+        component={(props) => <BlogPage globalStore={this.props} {...props} />}
+        />
```
Comment (kleberbaum) [inline src/Routes.js:44]: Would be nice to have a short description of the the destination.

File: src/assets/content/logo_main.svg [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<style type="text/css">
+	.st0{fill:#212529;}
+	.st1{fill:none;stroke:#000000;stroke-width:0.3;}
+	.st2{fill:#F61A42;}
+</style>
+<image style="display:none;overflow:visible;" width="5668" height="3514" id="logo_x5F_withtype_xA0_Image" xlink:href="EC8BD75C.png"  transform="matrix(0.24 0 0 0.24 0 0)">
+</image>
+<g>
+	<g>
+		<path class="st0" d="M541.9,349.6l-99.1,1.7c0,0,69.8,131,237.4,131s237.5-132.7,237.5-132.7h-99.1c0,0-52.2,49.4-138.4,49.4
```
Comment (kleberbaum) [inline src/assets/content/logo_main.svg:15]: We allow up to 119 characters as this is the width of GitHub code review;
I know it's a svg.

File: src/assets/content/logo_main.svg [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	<line class="st1" x1="388.8" y1="482.3" x2="424.8" y2="482.3"/>
+	<line class="st1" x1="442.8" y1="536.3" x2="442.8" y2="500.3"/>
+	<line class="st1" x1="917.6" y1="295.6" x2="917.6" y2="331.6"/>
+	<line class="st1" x1="971.6" y1="349.6" x2="935.6" y2="349.6"/>
+	<line class="st1" x1="971.6" y1="482.3" x2="935.6" y2="482.3"/>
+	<line class="st1" x1="917.6" y1="536.3" x2="917.6" y2="500.3"/>
+</g>
+<g>
+	<g>
+		<path class="st0" d="M489.4,288.2c0,0-32.9-48.7-28.5-119c4.5-70.3,42.2-119.7,42.2-119.7l-32.9-32.9c0,0-52.5,58.3-65.5,135.8
```
Comment (kleberbaum) [inline src/assets/content/logo_main.svg:29]: We allow up to 119 characters as this is the width of GitHub code review;
I know it's a svg.

File: src/assets/content/logo_main.svg [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+	<line class="st1" x1="346.8" y1="295.1" x2="382.8" y2="295.1"/>
+	<line class="st1" x1="400.8" y1="349.1" x2="400.8" y2="313.1"/>
+	<line class="st1" x1="503.1" y1="-37.3" x2="503.1" y2="-1.3"/>
+	<line class="st1" x1="557.1" y1="16.7" x2="521.1" y2="16.7"/>
+	<line class="st1" x1="557.1" y1="295.1" x2="521.1" y2="295.1"/>
+	<line class="st1" x1="503.1" y1="349.1" x2="503.1" y2="313.1"/>
+</g>
+<g>
+	<g>
+		<path class="st0" d="M870.7,288.2c0,0,32.9-48.7,28.5-119C894.7,98.9,857,49.6,857,49.6l32.9-32.9c0,0,52.5,58.3,65.5,135.8
```
Comment (kleberbaum) [inline src/assets/content/logo_main.svg:43]: We allow up to 119 characters as this is the width of GitHub code review;
I know it's a svg.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: src/Routes.js (hunk 1 above)
Comment (Aichnerc) [reply src/Routes.js:44]: The destinations are described above when they are imported.

File: src/assets/content/logo_main.svg (hunk 2 above)
Comment (Aichnerc) [reply src/assets/content/logo_main.svg:15]: Hell no.

File: src/assets/content/logo_main.svg (hunk 3 above)
Comment (Aichnerc) [reply src/assets/content/logo_main.svg:29]: Hell no.

File: src/assets/content/logo_main.svg (hunk 4 above)
Comment (Aichnerc) [reply src/assets/content/logo_main.svg:43]: Hell no.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**